File: scripts/services/ntt_gdc.py
from core.utils import make_service

SERVICE = {
    "id": "ntt-gdc",
    "name": "NTT GDC",
    "desc": "全球數據中心與園區基礎設施",
    "category": "datacenter",
    "page_url": "https://services.global.ntt/"
}

def fetch(ctx):
    # No stable anonymous incident-history feed is exposed for this service.
    # Keep the official destination available without falsely reporting a failure.
    return make_service(SERVICE, "restricted", "official", [])

File: assets/services/ntt-gdc.js
/* CloudStatus service module: NTT GDC */
(function () {
  "use strict";
  window.CloudStatusServices.register({
  "id": "ntt-gdc",
  "name": "NTT GDC（NTT 數據中心）",
  "desc": "全球數據中心與園區基礎設施",
  "category": "datacenter",
  "page": "https://services.global.ntt/",
  "parser": "ntt-gdc",
  "sources": [
    {
      "type": "reader",
      "label": "官方頁",
      "tier": 50,
      "kind": "official-status",
      "priority": 50
    }
  ],
  "publicStructuredFeed": false
});
})();
